src/include/duckdb/execution/operator/helper/physical_batch_collector.hpp

```diff
@@ -33,10 +33,6 @@ class PhysicalBatchCollector : public PhysicalResultCollector {
 		return true;
 	}
 
-	bool IsSink() const override {
-		return true;
```

Comment:
is this on puropose?

Reply:
Yea `PhysicalResultCollector` already overrides IsSink to true, so this is redundant as far as I can tell.